core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -434,12 +444,17 @@ int getSpeed( OSMWay way )
             } else 
             {
                 if (way.hasTag("bicycle", "designated") || way.hasTag("bicycle", "official"))
-                    speed = getHighwaySpeed("cycleway");
-                else                
-                    speed = PUSHING_SECTION_SPEED * 2;
-            }
-        }
-
+                {
```

Comment:
I'll think about this here. It is too long ago. I attempted to simplify it now but suddenly some totally unrelated tests fail, althouth the change did pass Travis when I created the PR.